### PR TITLE
Added Mac shortcuts to Evernote cheat sheet

### LINKED ISSUE #3252: Evernote Cheat Sheet: Should include Mac shortcuts
Perhaps they can be added as new sections to this cheat sheet. Either that or create a new cheat sheet, but that would require more specific triggering.

------
IA Page:  http://duck.co/ia/view/evernote_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @smitche60

### PR DESCRIPTION
**Pull request by smitche60**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ X] Improvement
    - [X ] Bug fix: **`{Evernote Cheat Sheet}: Fix 3252`**
    - [X ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.
Added Mac shortcuts to Evernote cheat sheet
###### Which issues (if any) does this fix?

Fixes #3252 - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/evernote_cheat_sheet
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @smitche60

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by zekiel** [comment]
@smitche60 thanks -- looks like it was pushed w/o a link to the original IA page: https://duck.co/ia/view/evernote_cheat_sheet  I've updated your original comment to mention that and your name as the maintainer (sorry for any extra notification that generated). 

This will deploy to beta and should be good to merge pending no issues w/ testing. Thanks for the update. I'll close #3252 when this is merged.

File: share/goodie/cheat_sheets/json/evernote.json [hunk 1]
```diff
@@ -1,172 +1,233 @@
...
+			"val": "Jump to Notebook"
+		}, {
+			"key": "[Shift] [Cmd] [J]",
+			"val": "Jump to Tag"
+		}, {
+			"key": "[Cmd] [|]",
+			"val": "Select Note List"
+		}, {
+			"key": "[Cmd] [[]",
+			"val": "Back"
```
Comment (zekiel) [inline share/goodie/cheat_sheets/json/evernote.json]: you'll need double-backslashes here and in the Forward example to express the single open bracket. http://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html#code-blocks-in-the-key-property

**Comment by zekiel** [comment]
Just one odd bit to be adjusted: 

The Back/Forward commands for mac aren't displaying correctly. Check out the cheat sheet syntax to adjust for single open/closed brackets. I left you some feedback in line to fix that.

**Comment by smitche60** [comment]
@zekiel Sorry about that- thanks for the additional documentation. I have added the fix for the Forward and Back shortcuts.

**Comment by edgesince84** [comment]
thanks for making those changes @smitche60 - we'll get this reviewed again quickly

**Comment by alohaas** [comment]
@smitche60 this LGTM 👍 
Thank you!

**Pull request merged**